transmogrifier/cli.py

```diff
@@ -65,9 +72,15 @@ def main(
     etl_version = get_etl_version()
     match etl_version:
         case 1:
+            if output_file is None:
+                message = "--output-file must be set when using ETL_VERSION=1"
+                raise RuntimeError(message)
             transformer.transform_and_write_output_files(output_file)
         case 2:
-            transformer.write_to_parquet_dataset(output_file)
+            if output_location is None:
+                message = "-o / --output-location must be set when using ETL_VERSION=2"
+                raise RuntimeError(message)
+            transformer.write_to_parquet_dataset(output_location)
```

Comment:
Once ETL version 1 is deprecated, will `--output-location` be configured with `required=False` in the CLI (and eventually remove the need for this check)?

Reply:
That's correct!  Removing feature flags from Transmog will probably be the most complex of feature flag removal... so some care and a fine tooth comb will need to be used for that.